examples/server.rs

```diff
@@ -220,8 +220,18 @@ fn generate_pixels(img: Image, big_endian: bool, rgb_order: (u8, u8, u8)) -> Vec
 #[async_trait]
 impl Server for ExampleServer {
     async fn get_framebuffer_update(&self) -> FramebufferUpdate {
+        let pixels_width = 1024;
+        let pixels_height = 768;
         let pixels = generate_pixels(self.display, self.big_endian, self.rgb_order);
-        let r = Rectangle::new(0, 0, 1024, 768, Box::new(RawEncoding::new(pixels)));
+        let r = Rectangle::new(
+            0,
+            0,
+            pixels_width,
+            pixels_height,
+            Box::new(RawEncoding::new(pixels)),
+        );
         FramebufferUpdate::new(vec![r])
     }
+
+    async fn keyevent(&self, _ke: KeyEvent) {}
```

Comment:
Need to update this to `key_event`

Reply:
(Which reminds me, gotta add CI to this repo at some point)

Reply:
done in 012ab7e